inventure/Fake_Dependency_Service
# sync repo with new features

- mock by different sets of query params
- add feature to chain mock responses
- add feature to verify request headers
- add feature to verify binary request content
- add feature to mock response headers
- enhance performance of service

![image](https://user-images.githubusercontent.com/44781950/156855359-ca97a5c5-25e0-4dd2-8c6d-51df324d4cdf.png)
![image](https://user-images.githubusercontent.com/44781950/156855429-66a879da-aae9-4351-afe6-b7750e05ee74.png)
